packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -247,28 +247,30 @@ class _WidgetInspectorState extends State<WidgetInspector>
 class InspectorSelection {
   /// Render objects that are candidates to be selected.
   ///
-  /// Tools may wish to iterate through the list of candidates
+  /// Tools may wish to iterate through the list of candidates.
   List<RenderObject> get candidates => _candidates;
-
+  List<RenderObject> _candidates = <RenderObject>[];
   set candidates(List<RenderObject> value) {
     _candidates = value;
     index = 0;
   }
 
-  List<RenderObject> _candidates = <RenderObject>[];
-
   /// Index within the list of candidates that is currently selected.
   int index = 0;
 
+  /// Set the selection to empty.
   void clear() {
     _candidates = <RenderObject>[];
     index = 0;
   }
 
+  /// Selected render object.
```

Comment:
Should probably mention that it comes `from the [candidate] list`, and that `setting or clearing [candidates], e.g. with [clear]` resets the selected render object. Also that it returns null if the selection is invalid.